templates/firefox-ios.xliff

```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
+        <source>Check Review Quality</source>
+        <note>Text for the analyzer button, from No Analysis Card View displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.NoAnalysisCard.BodyLabel.Title.v120" xml:space="preserve">
+        <source>To know whether this product’s reviews are reliable, check the review quality. It only takes about 60 seconds.</source>
+        <note>Text for the body label, from No Analysis Card View displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.NoAnalysisCard.HeadlineLabel.Title.v120" xml:space="preserve">
+        <source>No info about these reviews yet</source>
+        <note>Text for the Headline Label, from No Analysis Card View displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.NoAnalysisCard.InProgress.Body.Label.v120" xml:space="preserve">
+        <source>This could take about 60 seconds.</source>
+        <note>Description of the No Analysis Card when the analysis is in progress displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.NoAnalysisCard.InProgress.Headline.Title.v120" xml:space="preserve">
+        <source>Checking review quality</source>
+        <note>Headline of the No Analysis Card when the analysis is in progress displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.OptInCard.DisclaimerText.Title.v120" xml:space="preserve">
+        <source>By selecting “Yes, Try It” you agree to the %1$@ by %2$@:</source>
+        <note>Text for the disclaimer that appears beneath the rating image of the Shopping Experience Opt In onboarding Card (Fakespot). The first parameter will be replaced by the Fakespot app name and the second parameter by the company name of Mozilla.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.OptInCard.FirstParagraph.Title.v120" xml:space="preserve">
+        <source>See how reliable product reviews are on %1$@ before you buy. Review checker, an experimental feature from %2$@, is built right into the browser. It works on %3$@ and  %4$@, too.
+
+Using the power of %5$@ by %6$@, we help you avoid biased and inauthentic reviews. Our AI model is always improving to protect you as you shop.</source>
+        <note>Label for the first paragraph of the Shopping Experience Opt In onboarding Card (Fakespot). The first parameter will be the website the user is coming from when viewing this screen (default Amazon). The second parameter will be replaced by the app name. The third and fourth parameters will be the other two websites that are currently supported (Amazon, Best Buy or Walmart) besides the one used for the first parameter. The fifth and sixth parameter are the Fakespot app name and the company name of Mozilla.</note>
```

Comment:
Having so many parameters in one string has a good chance of creating confusion for localizers, resulting in awkward/wrong translations.
For browser and android, these are two [strings](https://transvision.mozfr.org/?recherche=See+how+reliable+product+reviews&repo=all_projects&sourcelocale=en-US&locale=fr&search_type=strings) (stopping at the 4th parameter).
I'd suggest to mirror that here too

Reply:
Will be fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16979

Reply:
PR will land for v121